packages/babel-preset-mapbox/index.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+module.exports = function preset(context, opts) {
+  opts = opts || [];
```

Comment:
How can the user pass in a `browserslist` value?

Reply:
So the idea is to get `browserslist` from the `.babelrc` or else use a default value.

As per our last discussion `babel-preset-mapbox` would provide a direct way for user to provide browserslist: 
```
{
   presets: [["babel-preset-mapbox", [
    "last 1 version",
    "> 1%",
    "maintained node versions",
    "not dead"
  ]]
}
```
or 
```
{
   presets: [["babel-preset-mapbox", [
   ["browserslist", ["last 1 version",
    "> 1%",
    "maintained node versions",
    "not dead"
  ]]]
}
```
(Note: this is not yet implemented)